XIVSlothCombo/Combos/DRG.cs

```diff
@@ -310,6 +337,29 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     }
                 }
 
+                //Dragon Sight Feature
+                if (canWeaveAbilities)
+                {
+                    if (IsEnabled(CustomComboPreset.DragoonDragonSightFeature))
+                    {
```

Comment:
Put both feature blocks inside the canweave condition, means you don't check twice.